client/modules/client_state_machine.js

```diff
@@ -222,22 +234,6 @@ define(function(require) {
       // Transition according to current state rules.
       this.state.nextModule(module);
     }
-    handleError(error) {
-      if (monitor.isEnabled()) {
-        monitor.update({client: {
-          event: error.toString(),
-          time: timeManager.now(),
-          color: [255, 0, 0]
-        }});
-      }
-      
-      logError(error);
-      // If we bubble up an error this far, we transition instantly back to a 
-      // blank screen.
```

Comment:
For debugging purposes, can we transition to a screen that shows the error?

Reply:
I think that this is a good idea, but it would require adding a prod/debug mode into the server, which doesn't exist yet. I'll file an issue with this: #167.